lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -47,6 +47,9 @@
       <a href="${tweet_action}" class="share">
         <span class="icon fa fa-twitter" aria-hidden="true"></span><span class="sr">${_("Tweet that you've enrolled in this course")}</span>
       </a>
+      <a href="${facebook_link}" class="share">
```

Comment:
This context var `facebook_link` was already being passed?

Reply:
look further up in this file...

Reply:
@caesar2164 I think the larger question is, are we actually setting `PLATFORM_FACEBOOK_ACCOUNT`?
I didn't know we had a platform Facebook account...